hack/create-cluster.sh

```diff
@@ -7,24 +7,27 @@ set -o pipefail
 
 function usage() {
   echo "This script starts a kube cluster by kind."
-  echo "Usage: hack/create-cluster.sh <CLUSTER_NAME> [KUBECONFIG]"
-  echo "Example: hack/create-cluster.sh host /root/.kube/karmada.config"
+  echo "Usage: hack/create-cluster.sh <CLUSTER_NAME> <POD_CIDR> <SERVICE_CIDR> [KUBECONFIG]"
```

Comment:
I don't like the way requires users provide CIDR.

Reply:
Modified.

Using environment variables instead. If user do not set，the script will generate a random CIDR.

Reply:
CIDR information in the cluster generated using the default Kind configuration:

```
  - command:
    - --cluster-cidr=10.244.0.0/16
    - --service-cluster-ip-range=10.96.0.0/16
```